apps/sertra/pages/server/[id].tsx

```diff
@@ -50,6 +52,10 @@ const ServerDetail: React.FC<{
   listing_data: ServerListingDetailData;
   mods_data: ListingMod[];
 }> = ({ listing_data, mods_data }) => {
+  const [showPopup, setShowPopup] = useState(false);
+  const togglePopup = () => {
```

Comment:
Now that there's just the one command in the function this could as well be `const togglePopup = () => setShowPopup((current) => !current);`